draft-ietf-quic-transport.md

```diff
@@ -486,9 +486,22 @@ Key Phase Bit:
   recipient of a packet to identify the packet protection keys that are used to
   protect the packet.  See {{QUIC-TLS}} for details.
 
+\[\[RFC editor: please remove the Google QUIC Demultipexing bit before
+publication.]]
```

Comment:
For clarity, we don't want them to actually eliminate the bit, just the text describing why it's there.  I'd just steal the Note from the changelog asking that "this section" be removed, and put it immediately after (rather than before) the header.